### DISCUSSION TITLE
Report based on Activity

### DISCUSSION THREAD

**Discussion by ddeimeke** [kimai/kimai#3069] [General]
It would be great to have a report based on global activities which are not bound to any one project.

There are a couple of activities I use in several projects like "Research" or "Podcasting" or "Support" and I would like to have an idea how much time I spend on such activities.

**Comment by kevinpapst** [comment]
Feature requests are created here: https://github.com/kevinpapst/kimai2/issues/new/choose

If you want to have a full new report in Kimai, please share MORE vision than "a new report based on global activities".
Feature requests should contain ideas and input for discussion, so please: add what exactly you would like to see.
Charts, tables, grouped by date/user/project/customer ...

**Comment by kevinpapst** [comment]
Duplicate of https://github.com/kevinpapst/kimai2/discussions/3068 - closing
